sphinx/config.py

```diff
@@ -51,6 +51,8 @@ class Config(object):
         # the real default is locale-dependent
         today_fmt = (None, 'env', [str]),
 
+        figure_language = (None, 'env', [str]),
+        figure_language_filename = ('%(root)s.%(language)s%(ext)s', 'env', [str]),
```

Comment:
> I didn't know about the default. But explicit type hinting is better for me.
> (it looks like other variables also uses [str] hinting)

Yes, but only when the default is not of type string.  The default value already contributes to the list of types a configuration value can take, so adding it here adds *no* value.

> This replaces relative path from docname, not document root.
> So it is difficult to refer shared image directories, now.

Ah, well, might leave it to another PR to actually do this then.